lib/Basics/process-utils.h

```diff
@@ -168,6 +168,14 @@ void TRI_CreateExternalProcess(char const* executable,
                                std::vector<std::string> additionalEnv,
                                bool usePipes, ExternalId* pid);
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief Reads from the pipe of processes
+////////////////////////////////////////////////////////////////////////////////
+
+bool TRI_ReadPipe(ExternalProcess const* process,
+                  char* buffer,
+                  size_t &bufferSize);
```

Comment:
```suggestion
                  size_t& bufferSize);
```
But as mentioned above, this API looks strange.